compiler/src/test/scala/dx/compiler/CompilerTest.scala

```diff
@@ -1119,8 +1119,32 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
     ignoreReuseFlag shouldBe Some(JsBoolean(true))
   }
 
+  it should "override job-reuse flag on applet" taggedAs NativeTest in {
+    val path = pathFromBasename("compiler", "add_runtime_hints.wdl")
+    val extrasContent =
+      """|{
+         |  "ignoreReuse": false
+         |}
+         |""".stripMargin
+    val retval = WithExtras(extrasContent) { extrasPath =>
+      val args = path.toString :: "--extras" :: extrasPath :: cFlags
+      Main.compile(args.toVector)
+    }
+    val appletId = retval match {
+      case SuccessfulCompileNativeNoTree(_, Vector(x)) => x
+      case other =>
+        throw new Exception(s"unexpected result ${other}")
+    }
+
+    // make sure the job reuse flag is set
+    val (_, stdout, _) =
+      SysUtils.execCommand(s"dx describe ${dxTestProject.id}:${appletId} --json")
+    val ignoreReuseFlag = stdout.parseJson.asJsObject.fields.get("ignoreReuse")
+    ignoreReuseFlag shouldBe Some(JsBoolean(false))
+  }
+
   it should "set job-reuse flag on workflow" taggedAs NativeTest in {
-    val path = pathFromBasename("subworkflows", basename = "trains_station.wdl")
+    val path = pathFromBasename("subworkflows", basename = "check_route.wdl")
```

Comment:
use this wdl instead which makes it easier to check if the flag is passed to embedded tasks.